library/src/main/java/com/devbrackets/android/playlistcore/service/PlaylistServiceCore.java

```diff
@@ -738,6 +738,20 @@ protected void performPlay() {
                 videoPlayer.play();
             }
         }
+        
+        //Immediately pauses the media
```

Comment:
Sorry for the delayed response.

The fix should probably be in the `startMediaPlayer(@NonNull MediaPlayerApi mediaPlayerApi)`.  Specifically [Line 954](https://github.com/brianwernick/PlaylistCore/blob/master/library/src/main/java/com/devbrackets/android/playlistcore/service/PlaylistServiceCore.java#L954) should handle the determination if we can start playing, or not.  This may have other repercussions though since I haven't taken the time to look in to it completely.  I should have a little time this weekend to look at it, or you can look in to it and update the fix.